src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -485,12 +485,12 @@ bool fx_muxer_t::resolve_hostpolicy_dir(
     {
         if (!pal::file_exists(get_app(fx_definitions).get_runtime_config().get_path()))
         {
-            trace::error(_X("Failed to run as a standalone app because there is no '%s' file. If this should be a portable app instead, add that file specifying the appropriate framework."),
+            trace::error(_X("Failed to run as a standalone app. If this should be a portable app, add the %s file specifying the appropriate framework."),
```

Comment:
I'm wondering if instead of `standalone` and `portable`, we should be using the same terms as used in documentation:  [https://docs.microsoft.com/en-us/dotnet/core/deploying/](https://docs.microsoft.com/en-us/dotnet/core/deploying/)

Reply:
We can either keep it like this for this issue and create a new issue for the rename, or change it now.

There's at least 3 other locations in this same file that use the standalone\portable terminology when logging.

I don't really like "framework dependent" because the word "shared" is not present - a "self contained" app still depends upon the "framework"...